projects/demo-integrations/cypress/tests/kit/slider/slider.spec.ts

```diff
@@ -3,22 +3,15 @@ import {
     SLIDER_PAGE_URL,
 } from '../../../support/shared.entities';
 
-const goToApiPage = () => {
-    cy.get('[tuitab]').eq(1).click();
-};
-
 describe('Slider', () => {
     beforeEach(() => {
         cy.viewport('macbook-13');
-        cy.goToDemoPage(SLIDER_PAGE_URL);
+        cy.goToDemoPage(`${SLIDER_PAGE_URL}/API?tuiMode=null&max=89&min=0&step=1&size=s`);
```

Comment:
will it work in our proprietary repo ?

Reply:
Other specs in main branch also use `API`

<img width="892" alt="image" src="https://user-images.githubusercontent.com/12021443/154266362-ab961a41-c068-43f6-b62e-3db67a7f554d.png">

<img width="789" alt="image" src="https://user-images.githubusercontent.com/12021443/154266447-11e3974c-3745-419f-adc4-02e582921c79.png">